src/c++/perf_analyzer/command_line_parser.h

```diff
@@ -135,7 +135,7 @@ struct PerfAnalyzerParameters {
   // requests are not outstanding to the triton server. If the overhead
   // percentage exceeds the threshold, a warning is displayed.
   //
-  double overhead_pct_threshold{50.0};
+  int overhead_pct_threshold{50};
```

Comment:
Allow double for non-whole-number percentages?

Reply:
Tim prefers integers for simplicity